eth/stagedsync/stage_log_index.go

```diff
@@ -440,17 +438,21 @@ func pruneLogIndex(logPrefix string, tx kv.RwTx, tmpDir string, pruneTo uint64,
 
 			for _, l := range logs {
 				for _, topic := range l.Topics {
-					topics[string(topic.Bytes())] = struct{}{}
+					if err := topics.Collect(topic.Bytes(), []byte{}); err != nil {
```

Comment:
probably pass "nil" as a second value will be more optimal (no allocations).

Reply:
make sense! thanks